project.yml

```diff
@@ -3,10 +3,10 @@ options:
   bundleIdPrefix: com.spruceid.mobile
 packages:
   SpruceIDMobileSdk:
-    url: https://github.com/spruceid/mobile-sdk-swift
+    #url: https://github.com/spruceid/mobile-sdk-swift
     # from: 0.0.10
-    revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
-    # path: "../mobile-sdk-swift"
+    #revision: 379b4a3d4550df6ab9da0e5be3406d8021dcd253
+    path: "../mobile-sdk-swift"
```

Comment:
Can we point to a remote version? If we merge with a fixed local version, it will cause some problems for others

Reply:
Good catch, thanks! Fixed.